java/ql/src/utils/flowtestcasegenerator/GenerateFlowTestCase.py

```diff
@@ -221,7 +273,7 @@ def copyfile(fromName, toFileHandle):
     # Make a test extension file
     with open(resultYml, "w") as f:
         models = "\n".join('      - [%s]' %
-                          modelSpecRow[0].strip() for modelSpecRow in supportModelRows)
+                           modelSpecRow[0].strip() for modelSpecRow in supportModelRows)
```

Comment:
Is this change intended?

Reply:
This change was made by my autoformatter